examples/beginner-examples/money-router/README.md

```diff
@@ -0,0 +1,42 @@
+# Advanced Sample Hardhat Project
```

Comment:
Please replace the README boilerplate with a high level description and any external links (if any)